### PR TITLE
[15.0][MIG] mrp_workorder_sequence

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Standard Migration
@ForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM, thanks for the test cases :+1:

File: mrp_workorder_sequence/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "MRP Work Order Sequence",
     "summary": "adds sequence to production work orders.",
-    "version": "14.0.1.0.1",
+    "version": "15.0.1.0.1",
```
Comment (AaronHForgeFlow) [inline mrp_workorder_sequence/__manifest__.py:7]: 15.0.1.0.0

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: code review. Minor comment

**Pull request merged**